v-next/hardhat-utils/src/fs.ts

```diff
@@ -343,7 +343,7 @@ export async function mkdir(absolutePath: string): Promise<void> {
  * Alias for `mkdir`.
  * @see mkdir
  */
-export const ensureDir = mkdir;
+export const ensureDir: (absolutePath: string) => Promise<void> = mkdir;
```

Comment:
can we make this `typeof mkdir` instead?

Reply:
I believe that would work. I have a slight preference for the explicit version tbh.

Reply:
Addressed in d3cb5d9e3